homeassistant/core.py

```diff
@@ -1257,6 +1265,65 @@ def as_dict(self) -> Dict:
             'config_source': self.config_source
         }
 
+    def _update(self, values: Dict, source: str) -> None:
```

Comment:
I prefer if update methods have each value as a keyword argument.

So it will be something like 

```
data = await store.async_load()
await hass.config.update(
  latitude=data['latitude'],
  # etc
)
```

Reply:
We do a similar thing for entity and device registry.

Reply:
Fixed.